Fix instructions for installing "shiny" library

CrackLord opened the pull request:
If you run what is stated in the readme you get:

```
> install.packages("Shiny")
Installing package into ‘/usr/local/lib/R/site-library’
(as ‘lib’ is unspecified)
Warning messages:
1: package ‘Shiny’ is not available (for R version 3.6.1) 
2: Perhaps you meant ‘shiny’ ? 
```

So it seems that it is case sensitive and all lowercase. Minor thing :)

CrackLord commented:
Actually improved it a bit by just adding a singular command to install all of the required libraries.

pmaji commented:
Thanks for this @CrackLord! I made one post-merge change that I failed to catch upon review. The "data" package should have been "data.table". Master is now up to date. Did you just feel like scratching the R itch haha? Always appreciate your contributions!

CrackLord commented:
Ah thanks.

> Did you just feel like scratching the R itch haha?

Actually I was looking at hosting it too, since in the README.md you said:

> Presently, the app is hosted here, although I recommend using this link sparingly, as I only have a small amount of free runtime allocated to this app via my shinyapps.io account.

😃 

I also saw your video from a while back about the whale tracker and saw your mention of me. Didn't know about that video at all, was a good interview.

pmaji commented:
Oh sweet! Well I'm glad you were looking around and stumbled onto this! Please feel free :)

And great! So glad to hear that. I very much enjoyed that interview. Now just to hope that the bull rally resumes and we get more of a reason to push for that brand of research again.

CrackLord commented:
> Oh sweet! Well I'm glad you were looking around and stumbled onto this! Please feel free :)

I'm now hosting a backup of the tool on https://finances.cracklord.com/. The whales service is also back online at https://whales.cracklord.com (an old stable version).

> Now just to hope that the bull rally resumes and we get more of a reason to push for that brand of research again.

We can only hope at this point :)

pmaji commented:
Marvelous! Thanks so much for doing that! 

On the whales service, do you have a link to the branch of the repo you're using that is old but stable? I have a few snapshots I point people to who are having trouble so they can revert back to the older version that is still usable, but some folks still have trouble, so it'd be great to be able to point them to the one you're using since that seems fine.

CrackLord commented:
I believe the commit that is active on my server is this one:

https://github.com/pmaji/crypto-whale-watching-app/commit/2b567987cb338802bc28702c96c22bde8d48d0c0

pmaji commented:
Great, thank you! I'll probably call that out / reference it whenever folks raise similar questions in the future.

pmaji commented:
BTW @CrackLord just an FYI that I updated the README for the crypto whale app to reference that older stable commit and also point to your newly hosting URL. Thanks for the help as always! I'm going to try to point some folks to it via twitter shortly.

CrackLord commented:
> BTW @CrackLord just an FYI that I updated the README for the crypto whale app to reference that older stable commit and also point to your newly hosting URL. Thanks for the help as always! I'm going to try to point some folks to it via twitter shortly.

Thanks, I have also added some external monitoring to the service, along with some fixes on the systemd service definition to try and ensure additional stability on the service.

pmaji commented:
That's great news!

P.S. here's the promised thread: https://twitter.com/ByPaulJ/status/1193625719689109509?s=20
